src/main/java/tictactoe/Context.java

```diff
@@ -4,47 +4,46 @@
 
 public class Context {
 
-	private Aktion splash;
-	private Aktion xTurn;
-	private Aktion oTurn;
-	private Aktion ende;
-	private Aktion currentState;
+	private Action splash;
+	private Action xTurn;
+	private Action oTurn;
+	private Action end;
+	private Action currentState;
 	private Board board;
 	
 	
 	public Context() {
 		this.splash = new Splash(this);
 		this.xTurn = new XTurn(this);
 		this.oTurn = new OTurn(this);
-		this.ende = new Ende(this);
+		this.end = new End(this);
 		this.board = new Board();
 
 		this.currentState = this.getSplash();
 	}
 
-
-	public void setCurrentState(Aktion s) {
+	public void setCurrentState(Action s) {
 		this.currentState = s;
```

Comment:
I would choose a more appropriate name for the parameter like 'action' or 'state'.

```suggestion
	public void setCurrentState(Action state) {
		this.currentState = state;
```

Reply:
Prefer action, but in this context state is fine.